internal/server/v1/firehose/crud.go

```diff
@@ -292,7 +292,7 @@ func (api *firehoseAPI) handleUpdate(w http.ResponseWriter, r *http.Request) {
 		updates.Configs.EnvVars[confStencilURL] = api.makeStencilURL(*schema)
 	}
 
-	cfgStruct, err := makeConfigStruct(&updates.Configs)
+	cfgStruct, err := makeConfigStruct(&updates.Configs, time.Time(existingFirehose.CreatedAt))
```

Comment:
@sudheerpal we need to add 24 hrs to current time. This would add it to the orignal time when this firehose was created.

I think we don't need this param in makeConfigStruct. Rather, we shall always add 24hrs to time.Now(). wdyt?